src/palindrom.js

```diff
@@ -341,13 +363,14 @@ const Palindrom = (() => {
           return;
         }
 
-        const m = {
-          statusText: 'WebSocket connection could not be made.',
-          readyState: this._ws.readyState,
-          url: upgradeURL
-        };
+        const message = [
+          `WebSocket connection could not be made`,
+          ` readyState: ${this._ws.readyState}`
+        ].join('\n');
```

Comment:
Why not use just multiline string?

Reply:
Because the whitespace on the left would be included.